packages/jaeger-ui-components/src/TraceTimelineViewer/VirtualizedTraceView.tsx

```diff
@@ -495,11 +510,16 @@ export class UnthemedVirtualizedTraceView extends React.Component<VirtualizedTra
     );
   }
 
+  scrollToTop = () => {
+    const { topOfExploreViewRef } = this.props;
+    topOfExploreViewRef?.current?.scrollIntoView();
+  };
+
   render() {
     const styles = getStyles();
     const { scrollElement } = this.props;
     return (
-      <div>
+      <div ref={this.topTraceViewRef}>
```

Comment:
Is this ref used anywhere? Wouldn't it be actually better to scroll to the top of the TraceView, not the top of Explore? Or maybe it would make sense to make it generic for Explore, so it can work with other panels not only Trace View panel?

Reply:
Updated. It feels like a cleaner transition for the scroll to go to the top of the panel, as users may scroll up anyway to see the editor controls. Other panels can hook into the ref to use it to scroll to the top of Explore.